doc/user_guide/profiling.rst

```diff
@@ -125,6 +125,10 @@ This is the code sequence which is created by PSyclone::
 PSyclone guarantees that different ProfileStart/End pairs have
 different ``ProfileData`` variables.
 
+By default PSyclone will generate appropriate names to specify a
+particular region (the "Module" and "Region" strings in the example
```

Comment:
Please mention the fact that the names are currently not very clever with a link to the associated issue.

Reply:
I've done this further down (just after the example) as it is more explicit there.

Reply:
Thank you, that warning is useful. However, what I meant was a reference to the issue that we're seeing in LFRic with the auto-generated names.

Reply:
Ah yes, apologies. Done now.